codetransformer/transformers/literals.py

```diff
@@ -47,17 +47,17 @@ class overloaded_dicts(CodeTransformer):
     >>> f()
     OrderedDict([('a', 1), ('b', 2), ('c', 3)])
     """
-    def __init__(self, astype):
+    def __init__(self, tnfm):
```

Comment:
IMO, `astype` implies that the callable should be a type. `tnfm` I think is slightly clearer that it's just a callable